src/dymaptic.GeoBlazor.Core/Components/PortalFolder.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dymaptic.GeoBlazor.Core.Components;
+public class PortalFolder : MapComponent
```

Comment:
needs class-level xml comments

Reply:
Is this class used? I can't find any references.

Reply:
I think it is referenced when I was trying to get some things workings, but I removed it